### PR TITLE
MAINT-28153: Add rest service in order to create multiple memberships at once

### PR DESCRIPTION
**Pull request by mdiha**
Make sure to handle the possibility to add multiple memberships in the Vue component and add the appropriate rest service.

### CODE REVIEW

**Review by ahamdi — DISMISSED** [review]

File: component/portal/src/main/java/org/exoplatform/portal/rest/GroupRestResourcesV1.java [hunk 1]
```diff
@@ -408,6 +408,68 @@ public Response createMembership(MembershipImpl membership) throws Exception {
     return Response.noContent().build();
   }
 
+  @POST
+  @Path("multiMemberships")
+  @RolesAllowed("administrators")
+  @Consumes(MediaType.APPLICATION_JSON)
+  @ApiOperation(
+          value = "Creates a new membership",
```
Comment (azayati) [inline component/portal/src/main/java/org/exoplatform/portal/rest/GroupRestResourcesV1.java]: Modify this value
```suggestion
          value = "Creates new memberships",
```

**Review by azayati — CHANGES_REQUESTED** [review]
See review

File: component/portal/src/main/java/org/exoplatform/portal/rest/GroupRestResourcesV1.java (hunk 1 above)
Comment (mdiha) [reply component/portal/src/main/java/org/exoplatform/portal/rest/GroupRestResourcesV1.java]: Okay thanks

File: component/portal/src/main/java/org/exoplatform/portal/rest/GroupRestResourcesV1.java [hunk 2]
```diff
@@ -408,6 +408,68 @@ public Response createMembership(MembershipImpl membership) throws Exception {
     return Response.noContent().build();
   }
 
+  @POST
+  @Path("multiMemberships")
```
Comment (azayati) [inline component/portal/src/main/java/org/exoplatform/portal/rest/GroupRestResourcesV1.java]: IMO it is better to user this path:
```suggestion
  @Path("memberships/multiple")
```
cc @ahamdi WDYT ?

**Review by azayati — CHANGES_REQUESTED** [review]
See review

File: component/portal/src/main/java/org/exoplatform/portal/rest/GroupRestResourcesV1.java (hunk 2 above)
Comment (azayati) [reply component/portal/src/main/java/org/exoplatform/portal/rest/GroupRestResourcesV1.java]: Reference: https://community.exoplatform.com/portal/g/:spaces:engineering/engineering/wiki/REST_CRUD_APIs

**Review by ahamdi — APPROVED** [review]
LGTM

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**